### ISSUE TITLE
Level 11 - too many var names?

### ISSUE THREAD

**Issue by bakerfranke** [code-dot-org/dance-party#148]
On level 11, the Cat and Robot are named "left_dancer" and "right_dancer" respectively.  But in an HoC context the instructions to "make the Robot drop .... make the Robot high clap" might be too hard to follow since as a (kid) user I'm scanning the purple blocks for dance moves -- which only show e.g. `right_dancer`.  I can probably put 2 and 2 together, but would it be terrible if we just called the variables `cat_dancer` and `robot_dancer`?

**Comment by ryansloan** [comment]
Per @mrjoshida This is by design, we decided to show different names to demonstrate that you can give sprites names
